srcpkgs/openssh/INSTALL.msg

```diff
@@ -1,3 +1,4 @@
 CAUTION: After updating from OpenSSH <=8.1 to OpenSSH 8.2, the existing sshd
 will not allow new connections.  **Restart OpenSSH immediately** after this
 update to ensure you still can log in!
+Install 'openssh-sk-helper' to use FIDO authenticators on the client.
```

Comment:
I don't see the use of this line

Reply:
It used to work before and now it needs a separate package, sounds like something we should tell the user.

Reply:
Didn't you add this in the last few weeks?